app/stacks/uk-west/appeals-service/app-service.tf

```diff
@@ -1,6 +1,11 @@
 module "app_services" {
   source = "../../../components/appeals-app-services"
 
+  providers = {

```

Comment:
I have done this in a few places when testing things in that I moved this block to the top of the file. Can move back towards the bottom

Reply:
helps to reduce changes if we leave where it is I think, keeps a cleaner history?